lambdatimes/src/components/Content/Content.js

```diff
@@ -37,20 +45,25 @@ export default class Content extends Component {
           of the items from cardData. 
         - else, it should only return those cards whose 'tab' matched this.state.selected.
     */
-    return this.state.cards;
-  };
+		const displayedCards = this.state.cards.filter((card) => card.tab === this.state.selected);
+		if (this.state.selected === 'all') {
```

Comment:
I would switch the sequence of line 48 and 49 . You want, your code to read like english. you want to group your concerns.